CONTRIBUTING.md

```diff
@@ -97,17 +97,16 @@ you can enable the docker tests by setting a gradle property of
 ``"enable.docker.tests"`` to true. See the gradle.properties file in the root of the project
 for more details.
 
-2. From the root project directory, initialize repository dependencies
+2. Clone the repository recursively
 
-   `make init-git-submodules`
+    `git clone git@github.com:<username>/opentelemetry-java.git --recursive`
```

Comment:
```suggestion
    `git clone https://github.com/open-telemetry/opentelemetry-java.git --recursive`
```

Reply:
@anuraaga are you making code change suggestions to yourselves? :)

Reply:
Yeah I think it's an easy way to get a oneliner in :)